main.tf

```
@@ -40,7 +42,26 @@ resource "lacework_integration_aws_agentless_scanning" "lacework_cloud_account"
   bucket_arn                = aws_s3_bucket.agentless_scan_bucket[0].arn
   credentials {
     role_arn    = aws_iam_role.agentless_scan_cross_account_role[0].arn
-    external_id = random_string.external_id.result
+    external_id = local.external_id
+  }
+}
+
+resource "lacework_integration_aws_org_agentless_scanning" "lacework_cloud_account" {
+  // If var.organization is used then also add monitored accounts and scanning account as the caller.
+  count                     = var.global && local.is_org_integration ? 1 : 0
+  name                      = var.lacework_integration_name
+  scan_frequency            = var.scan_frequency_hours
+  query_text                = var.filter_query_text
+  scan_containers           = var.scan_containers
+  scan_host_vulnerabilities = var.scan_host_vulnerabilities
+  account_id                = data.aws_caller_identity.current.account_id
+  bucket_arn                = aws_s3_bucket.agentless_scan_bucket[0].arn
+  monitored_accounts        = var.monitored_accounts
+  management_account        = var.management_account
```

May need to update this based on feedback about the input variables.
```suggestion
  monitored_accounts        = var.organization.monitored_accounts
  management_account        = var.organization.management_account
```